postreise/plot/plot_interconnection_map.py

```diff
@@ -1,249 +1,179 @@
-import numpy as np
 import pandas as pd
-from bokeh.models import ColumnDataSource, HoverTool
 from bokeh.plotting import figure
-from bokeh.tile_providers import Vendors, get_provider
 from powersimdata.network.model import ModelImmutables
 from powersimdata.utility import distance
 
-from postreise.plot.plot_states import get_state_borders
-from postreise.plot.projection_helpers import project_borders, project_branch
+from postreise.analyze.check import _check_grid
+from postreise.plot.plot_states import plot_states
+from postreise.plot.projection_helpers import project_branch
 
 
 def count_nodes_per_state(grid):
     """Count nodes per state to add as hover-over info in :func`map_interconnections`
 
     :param powersimdata.input.grid.Grid grid: grid object.
-    :return: (*pandas.DataFrame*) -- dataframe containing state names and count of nodes per state.
+    :return: (*pandas.DataFrame*) -- index: state names, column: number of nodes.
     """
     id2state = ModelImmutables(grid.get_grid_model()).zones["id2abv"]
     grid.bus["state"] = grid.bus["zone_id"].map(id2state)
-    liststates = grid.bus["state"].value_counts()
-    state_counts = pd.DataFrame(liststates)
-    state_counts.reset_index(inplace=True)
-    state_counts.rename(columns={"index": "state", "state": "count"}, inplace=True)
+    counts = pd.DataFrame(grid.bus["state"].value_counts())
+    counts.reset_index(inplace=True)
+    counts.rename(columns={"index": "state", "state": "count"}, inplace=True)
 
-    return state_counts
+    return counts
 
 
 def map_interconnections(
-    grid, state_counts, hover_choice, hvdc_width=1, us_states_dat=None
+    grid,
+    branch_distance_cutoff=5,
+    figsize=(1400, 800),
+    branch_width_scale_factor=0.0005,
+    hvdc_width_scale_factor=0.001,
+    b2b_size_scale_factor=0.05,
```

Comment:
Should we change these to more natural values? Based on the other PR, we could do `0.5, 1.0, 50`.

Reply:
Oops. I thought I got all of them. Will normalize them.

Reply:
Done